design/README.md

```diff
@@ -1,34 +1,30 @@
 # Power Xpert Blue Icons
 
-This repository contains icon files for Power Xpert Blue 2.0+.
+This repository contains icon files for Power Xpert Blue. These icons are grouped into 'family' folders of related icons. Family folders follow a similar naming convention to Material Design, e.g., **Status** and **Devices**. Each icon should be available as a clean, minified SVG file. You may optionally include a source (AI) file for the icon as well - it may be beneficial to group families of icons into a single AI file.
 
-Each icon has its own folder which includes the clean SVG file and the source design file(s).
+## Icon Approval
 
-Each icon folder will be nested into a "family" folder. Family folders follow a pattern similar to MD, e.g., **Status** and **Devices**.
+All icons created for PX Blue must go through the Pull Request process and be approved by the PX Blue team before they will be merged into the master branch and released in the PX Blue icon packages. 
 
-## Design Approval
+If you are comfortable using GitHub and the command line on your computer, you can follow the instructions below for creating your own Pull Request. 
 
-Designers can freely create icons for their project in their own feature branch, but in order to incorporate into the design system a pull request must be created and approved. See Git Steps for Designers below for more information.
+> If you're not familiar with GitHub and/or the command line, you can also send us your icons directly via email and we will create the Pull Request for you and relay any feedback.
```

Comment:
should we include the email address here? Or we don't want the crawlers to index our email for malicious purposes?

Reply:
I left it out to try to cut down on crawler spam